pydiscourse/client.py

```diff
@@ -655,19 +722,46 @@ def category(self, name, parent=None, **kwargs):
 
         return self._get(u'/category/{0}.json'.format(name), **kwargs)
 
-    def site_settings(self, **kwargs):
+    def delete_category(self, category_id, **kwargs):
         """
 
         Args:
+            category_id:
             **kwargs:
 
         Returns:
 
         """
-        for setting, value in kwargs.items():
+        return self._delete(u'/categories/{0}'.format(category_id), **kwargs)
+
+    def site_settings(self, settings, **kwargs):
```

Comment:
This changes the function interface, adding a new required variable `settings`.

Reply:
My mistake, I was trying to keep those changes separate. I've added 006b7d4 which reverts the change to site_settings.